client/src/partials/patient_edit/edit_patient.html

```diff
@@ -41,7 +41,7 @@
                   <div class="form-group" >
                     <label for="patient-dob" class="col-xs-3 control-label">{{ "PATIENT_REG.DOB" | translate }}</label>
                     <div class="col-xs-9">
-                      <input required id="patient-dob" type="date" class="form-bhima" ng-model="patient.dob" min="1900-01-01" max="{{ getMaxDate() }}">
+                      <input required id="patient-dob" type="date" class="form-bhima" ng-model="patient.dob" min="{{ getMinDate() }}" max="{{ getMaxDate() }}">
```

Comment:
Ok, I updated the pull request as requested.  I just put them in $scope.  Let me know if you prefer them in session.